README.md

```diff
@@ -1383,111 +1387,439 @@ This section contains detailed documentation for the following aspects of Terrag
+      "import",
+      "push",
+      "refresh"
+    ]
+
+    arguments = [
+      "-var-file=${include.parent}/common.tfvars"
+    ]
+  }
 }
+```
 
-inputs = {
-  aws_region  = local.aws_region
-  s3_endpoint = "com.amazonaws.${local.aws_region}.s3"
+The common.tfvars located in the terraform root folder will be included by all applications, whatever their relative location to the root.
+
+##### `include.child`
+
+This value is a direct replacement for the `get_terragrunt_dir()` function.
+
+`include.child` returns the directory where the Terragrunt configuration file (by default `terragrunt.hcl`) lives.
+This is useful when you need to use relative paths with [remote Terraform
+configurations](#remote-terraform-configurations) and you want those paths relative to your Terragrunt configuration
+file and not relative to the temporary directory where Terragrunt downloads the code.
+
+For example, imagine you have the following file structure:
+
+```
+/terraform-code
+├── common.tfvars
+├── frontend-app
+│   └── terragrunt.hcl
+```
+
+Inside of `/terraform-code/frontend-app/terragrunt.hcl` you might try to write code that looks like this:
+
+```hcl
+terraform {
+  source = "git::git@github.com:foo/modules.git//frontend-app?ref=v0.0.3"
+
+  extra_arguments "custom_vars" {
+    commands = [
+      "apply",
+      "plan",
+      "import",
+      "push",
+      "refresh"
+    ]
+
+    arguments = [
+      "-var-file=../common.tfvars" # Note: This relative path will NOT work correctly!
+    ]
+  }
 }
 ```
 
-You can use any valid terragrunt expression in the `locals` configuration. The `locals` block also supports referencing other `locals`:
+Note how the `source` parameter is set, so Terragrunt will download the `frontend-app` code from the `modules` repo
+into a temporary folder and run `terraform` in that temporary folder. Note also that there is an `extra_arguments`
+block that is trying to allow the `frontend-app` to read some shared variables from a `common.tfvars` file.
+Unfortunately, the relative path (`../common.tfvars`) won't work, as it will be relative to the temporary folder!
+Moreover, you can't use an absolute path, or the code won't work on any of your teammates' computers.
+
+To make the relative path work, you need to use `include.child` to combine the path with the folder where
+the `terragrunt.hcl` file lives:
 
+```hcl
+terraform {
+  source = "git::git@github.com:foo/modules.git//frontend-app?ref=v0.0.3"
+
+  extra_arguments "custom_vars" {
+    commands = [
+      "apply",
+      "plan",
+      "import",
+      "push",
+      "refresh"
+    ]
+
+    # With the include.child value, you can use relative paths!
+    arguments = [
+      "-var-file=${include.child}/../common.tfvars"
+    ]
+  }
+}
 ```
-locals {
-  x = 2
-  y = 40
-  answer = local.x + local.y
+
+For the example above, this path will resolve to `/terraform-code/frontend-app/../common.tfvars`, which is exactly
+what you want.
+
+
+##### `include.relative`
```

Comment:
This should be named `path_relative_to_include`, so the direction is clear in the name, and it parallels the existing function.